docs/data-sources/antiddos.md

```diff
@@ -21,6 +21,8 @@ data "huaweicloud_antiddos" "antiddos" {
 ## Argument Reference
 The following arguments are supported:
 
+* `region` - (Optional) The region in which to obtain the Antiddos client.  If omitted, the provider-level region will work as default. Changing this creates a new resource.
```

Comment:
s/Changing this creates a new resource.//

Reply:
This sentence has been removed in datasource docs.